perun/postprocess/regressogram/methods.py

```diff
@@ -113,29 +113,6 @@ def regressogram(x_pts, y_pts, statistic_function, buckets):
         )
     }
 
-
-def render_step_function(graph, x_pts, y_pts, graph_params):
```

Comment:
What is the issue with this function?

Reply:
Nothing really, the function was just moved to the scatter/factory.py module as it contains code related to plotting and not computing the postprocessing model itself.

Reply:
Found her later, fine decision.